stable/velero/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+Check that the velero is up and running:
```

Comment:
I would add the follow here so that users can actually verify.

```
Check that the velero is up and running:

    kubectl get deployment/{{ .Release.Name }} -n {{ .Release.Namespace }}

Check that the secret has been created:

    kubectl get secret/{{ include "velero.fullname" . }} -n {{ .Release.Namespace }}
```

Reply:
Done